week-1/exercises/initials.js

```diff
@@ -3,4 +3,11 @@ let middleName = "Katherine";
 let lastName = "Johnson";
 
 // Declare a variable called initials that stores the first character of each string in upper case to form the user's initials
-// Log the variable in each case
+// Log the variable in each case.
+//Here we use the charAt(0) method with  the toUpperCase() method. (the first character of a string in JavaScript starts with 0)
+let initials =
+  firstName.charAt(0).toUpperCase() +
+  middleName.charAt(0).toUpperCase() +
+  lastName.charAt(0).toUpperCase();
```

Comment:
This is correct & gets the right answer. Good use of 'toUpperCase' for completeness, but is it perhaps redundant in this situation?